sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcSchemaIOProvider.java

```diff
@@ -174,6 +174,12 @@ protected JdbcIO.DataSourceConfiguration getDataSourceConfiguration() {
                 .collect(Collectors.toList());
         dataSourceConfiguration = dataSourceConfiguration.withConnectionInitSqls(initSqls);
       }
+
+      @Nullable Integer maxConnections = config.getInt32("maxConnections");
```

Comment:
we should tolerate the incoming schema not containing this field for backward compatibility